src/main/webapp/creatives/maps/simid_map_creative.js

```diff
@@ -0,0 +1,54 @@
+
+import BaseSimidCreative from '../base_simid_creative.js';
+
+/**
```

Comment:
Move this to just before the class definition, and end it with a period

Reply:
fixed